wp-content/plugins/aliemu-plugins/inc/styles/colors.styl

```diff
@@ -1,4 +1,9 @@
 $primary          = #00B092
+$blue1            = #0A4A77
+$blue2            = #082254
+$blue3            = #040428
+$orange           = #FF884C
```

Comment:
I'll probably rename these to `primary`, `color1` ... `color4` in the event we ever change the scheme again